### DISCUSSION TITLE
Installation PowerShell via MS-Store unter Windows 10 Pro v21H1

### DISCUSSION THREAD

**Discussion by helmutkilian** [TobiasPSP/OReilly#3] [Allgemeines]
Vorab, falls mir hier etwas misslingen sollte: Ich bin neu bei Github.

Ich arbeite immer mit einem lokalen Windows-Benutzerkonto. Im Buch auf S. 25 oben steht, man könne PowerShell aus dem Store installieren, ohne sich am Store mit einem MS-Konto anzumelden. Das funktioniert bei mir nicht. Wenn man das Anmeldefenster wegklickt, passiert nichts. Ich kam um eine Anmeldung am Store nicht vorbei, um PowerShell aus dem Store zu installieren.

**Comment by TobiasPSP** [comment]
Vielen Dank für die Info. Bei meinen Tests konnte die Anmeldung übersprungen werden. Möglicherweise hat sich das geändert. Ich werde das überprüfen.
